jbpm-benchmarks/kieserver-performance-tests/src/main/resources/logback.xml

```diff
@@ -67,4 +67,9 @@
     <level value="debug" />
     <appender-ref ref="asyncDebugConsoleAppender" />
   </logger>
+
+  <logger name="org.jbpm.test.performance.taskassigning">
+    <level value="debug" />
+    <appender-ref ref="asyncDebugConsoleAppender" />
+  </logger>
```

Comment:
Do you plan to leave this on debug?

Reply:
I do; there are not so many messages, but in case the scenario gets stuck, they are priceless.